### PR TITLE
Update CVE-2022-0419.json

### PR DESCRIPTION
**Pull request by JamieSlome**

### CODE REVIEW

File: 2022/0xxx/CVE-2022-0419.json [hunk 1]
```diff
@@ -1,88 +1,89 @@
...
+        {
+          "product": {
+            "product_data": [
+              {
+                "product_name": "radareorg/radare2",
+                "version": {
+                  "version_data": [
+                    {
+                      "version_affected": "<",
+                      "version_value": "6.0.0"
```
Comment (trufae) [inline 2022/0xxx/CVE-2022-0419.json:20]: Should be 5.6.0

File: 2022/0xxx/CVE-2022-0419.json [hunk 2]
```diff
@@ -1,88 +1,89 @@
...
\ No newline at end of file
+  },
+  "data_format": "MITRE",
+  "data_type": "CVE",
+  "data_version": "4.0",
+  "description": {
+    "description_data": [
+      {
+        "lang": "eng",
+        "value": "NULL Pointer Dereference in GitHub repository radareorg/radare2 prior to 6.0.0."
```
Comment (trufae) [inline 2022/0xxx/CVE-2022-0419.json:39]: Should be 5.6.0

Comment (JamieSlome) [reply 2022/0xxx/CVE-2022-0419.json:39]: @trufae - please see https://github.com/CVEProject/cvelist/pull/4434

File: 2022/0xxx/CVE-2022-0419.json (hunk 1 above)
Comment (JamieSlome) [reply 2022/0xxx/CVE-2022-0419.json:20]: @trufae - please see https://github.com/CVEProject/cvelist/pull/4434

**Pull request merged**